.travis.yml

```diff
@@ -2,10 +2,20 @@ language: python
 python: 3.7
 install: "pip3 install flake8 mypy==0.761 PyYAML psycopg2 SQLAlchemy testing.postgresql sqlalchemy-stubs paramiko freezegun docker"
 script:
+  - sudo groupadd jobadder
+  - sudo usermod -a -G jobadder travis
+  - ssh-keygen -f ~/.ssh/id_rsa -P ""
```

Comment:
Please specify more clearly that we want an RSA key in a specific format, otherwise on some very recent distributions like F31 the generated key is not recognized as valid by paramiko. We need:

`ssh-keygen -f ~/.ssh/id_rsa -P "" -t rsa -m PEM`